Add info to Spec: Group page (#4877)
segmentio/segment-docs: src/connections/spec/group.md

title: 'Spec: Group'
---

The `group` API call is how you associate an individual user with a group, such as a company, organization, account, project, or team.

> The `group` call enables you to identify what account or organization your users are part of. There are two IDs that are relevant in a `group` call: the `userId`, which belongs and refers to the user, and the `groupId`, which belongs and refers to the specific group. A user can be in more than one group which would mean different `groupId`s, but the user will only have one `userId` that is associated to each of the different groups. Kkeep in mind that not all platforms support multiple groups for a single user. 

Reviewer: ```suggestion
The `group` call enables you to identify what account or organization your users are part of. There are two IDs that are relevant in a `group` call: the `userId`, which belongs and refers to the user, and the `groupId`, which belongs and refers to the specific group. A user can be in more than one group which would mean different `groupId`s, but the user will only have one `userId` that is associated to each of the different groups. Keep in mind that not all platforms support multiple groups for a single user. 
```

> The `group` API call is how you associate an individual user with a group—be it a company, organization, account, project, team or whatever other crazy name you came up with for the same concept!

Reviewer: ```suggestion
The `group` API call is how you associate an individual user with a group, such as a company, organization, account, project, or team.
```

> The `group` call is one of our methods that will enable you to identify what account or organization your users are part of. There are two IDs that are relevant in a `group` call: the `userId`, which belongs and refers to the user, and the `groupId`, which belongs and refers to the specific group. A user can be in more than one group which would mean different `groupId`s, but the user will only have one `userId` that is associated to each of the different groups (keep in mind that not all platforms support multiple groups for a single user). 

Reviewer: ```suggestion
The `group` call enables you to identify what account or organization your users are part of. There are two IDs that are relevant in a `group` call: the `userId`, which belongs and refers to the user, and the `groupId`, which belongs and refers to the specific group. A user can be in more than one group which would mean different `groupId`s, but the user will only have one `userId` that is associated to each of the different groups. Kkeep in mind that not all platforms support multiple groups for a single user. 
```

{% include components/reference-button.html href="https://university.segment.com/introduction-to-segment/324252?reg=1&referrer=docs" icon="media/academy.svg" title="Segment University: The Segment Methods" description="Check out our high-level overview of these APIs in Segment University. (Must be logged in to access.)" %}

> In addition to the groupId, which is how you'd identify the specific group or company, the group method can also receive traits that are specific to the group, like industry or number of employees for example, that will belong to that specific "account". And, like the traits of an identify call, these can be updated when you call the same trait with a different value.

Reviewer: ```suggestion
In addition to the `groupId`, which is how you'd identify the specific group or company, the group method receives traits that are specific to the group, like industry or number of employees for example, that belong to that specific account. Like the traits of an identify call, you can update these when you call the same trait with a different value.
```

> Calling `group` is a slightly more advanced feature, but it's helpful if you have accounts with multiple users.

Reviewer: ```suggestion
When using the `group` call, it's helpful if you have accounts with multiple users.
```
